<!--#include file="header.hidden" -->

<script>
  $("#link_publications").css("background-color", "#cacaca");
</script>

  <div class="container">
    <br>
    <p class="font-weight-light">

    <i class="far fa-newspaper"></i>&nbsp;<strong>Selected Publications</strong>

    <br>Complete publications at <a href='https://scholar.google.com/citations?hl=en&user=Mzz8rBAAAAAJ&view_op=list_works&sortby=pubdate'>Google Scholar</a>

    <p class="font-weight-light">
    <i class="far fa-dot-circle"></i>&nbsp;Currently I am continuing work on expressRNA and helping develop microbeAtlas, an interactive fast interface to the analysis of more than 1M metagenomic samples. I also try to maintain several collaborations on RNA bioinformatics. Main publications from this period are:
    </p>

    <div style='padding-left: 15px;'>

    <blockquote class="blockquote font-weight-light">
      Miha Modic, Markus Grosch, Gregor Rot, Silvia Schirge, Tjaša Lepko, Tomohiro Yamazaki, Flora C.Y. Lee, Ejona Rusha, Dmitry Shaposhnikov, Michael Palo, Juliane Merl-Pham, Davide Cacchiarelli, Boris Rogelj, Stefanie M. Hauck, Christian von Mering, Alexander Meissner, Heiko Lickert, Tetsuro Hirose, Jernej Ule, Micha Drukker
      <br><a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30270-9?fbclid=IwAR1m0xoJAlAqO9Z8rm8z5KNezFwsUMVOpWa2cf0oj1kQ7lkB0G0OwYNNf4g">Cross-Regulation Between TDP-43 and Paraspeckles Promotes Pluripotency-Differentiation Transition</a>
      <footer class="blockquote-footer">Molecular Cell (2019), News coverage: <a href='https://www.rtvslo.si/znanost-in-tehnologija/ijs-prebojna-studija-za-razumevanje-procesov-diferenciacije-celic/487313'>RTV Slovenia</a>, <a href='https://www.delo.si/novice/znanoteh/slovenska-ekipa-s-prebojno-studijo-o-zgodnjem-razvoju-celic-177970.html'>Delo</a></footer>
    </blockquote>

    </div>

    <p class="font-weight-light">
      <i class="far fa-dot-circle"></i>&nbsp;During my Phd I developed integrative approaches to study post-transcriptional modifications. We combined regulatory information with RNA-protein binding iCLIP data. In this way, we can visualize and represent regulatory information on exon/polyA site usage in relation to RNA-protein binding. You can read more about this and other work in these publications:
    </p>

    <div style='padding-left: 15px;'>

    <blockquote class="blockquote font-weight-light">
      Gregor Rot, Zhen Wang, Ina Huppertz, Miha Modic, Tina Lenče, Martina Hallegger, Nejc Haberman, Tomaž Curk, Christian von Mering, Jernej Ule
      <br><a href="http://www.cell.com/cell-reports/abstract/S2211-1247(17)30522-3">High-resolution RNA maps suggest common principles of splicing and polyadenylation regulation by TDP-43</a>
      <footer class="blockquote-footer">Cell Rep (2017)</footer>
    </blockquote>

    <blockquote class="blockquote font-weight-light">
      Matteo Cereda, Uberto Pozzoli, Gregor Rot, Peter Juvan, Anthony Schweitzer, Tyson Clark, Jernej Ule
      <br><a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-1-r20">RNAmotifs: prediction of multivalent RNA motifs that control alternative splicing</a>
      <footer class="blockquote-footer">Genome Biology (2014)</footer>
    </blockquote>

    </div>


    <p class="font-weight-light">
      <i class="far fa-dot-circle"></i>&nbsp;In May 2008 I started working on several bioinformatics projects at the Faculty of Computer and Information Science in Ljubljana. We developed dictyExpress, probably the most advanced web application for gene expression profile analytics (clustering, GO enrichment analysis) at the time. I co-developed the first analysis <a href='https://github.com/tomazc/iCount' target='_new'>pipeline for iCLIP</a> RNA-protein binding data and started working on analysis of post-transcriptional modifications. Publications from this period include:
    </p>

    <div style='padding-left: 15px;'>

    <blockquote class="blockquote font-weight-light">
    Julian Konig, Kathi Zarnack, Gregor Rot, Tomaž Curk, Melis Kayikci, Blaž Zupan, Daniel J Turner, Nicholas M Luscombe, & Jernej Ule
    <br><a href="https://www.nature.com/articles/nsmb.1838">iCLIP reveals the function of hnRNP particles in splicing at individual nucleotide resolution</a>
    <footer class="blockquote-footer">Nature Structural & Molecular Biology (2010)</footer>
    </blockquote>

    <blockquote class="blockquote font-weight-light">
    Boris Rogelj, Laura E. Easton, Gireesh K. Bogu, Lawrence W. Stanton, Gregor Rot, Tomaž Curk, Blaž Zupan, Yoichiro Sugimoto, Miha Modic, Nejc Haberman, James Tollervey, Ritsuko Fujii, Toru Takumi, Christopher E. Shaw, Jernej Ule
    <br><a href="https://www.nature.com/articles/srep00603">Widespread binding of FUS along nascent RNA regulates alternative splicing in the brain</a>
    <footer class="blockquote-footer">Nature Scientific Reports (2012)</footer>
    </blockquote>

    <blockquote class="blockquote font-weight-light">
    Tomaž Curk, Gregor Rot, Blaž Zupan
    <br><a href="https://academic.oup.com/nar/article/39/suppl_2/W444/2506138">SNPsyn: detection and exploration of SNP-SNP interactions</a>
    <footer class="blockquote-footer">Nucleic Acids Research (2011)</footer>
    </blockquote>

    <blockquote class="blockquote font-weight-light">
    Gregor Rot, Anup Parikh, Tomaž Curk, Adam Kuspa, Gad Shaulsky, Blaž Zupan
    <br><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-265">dictyExpress: a Dictyostelium discoideum gene expression database with an explorative data analysis web interface</a>
    <footer class="blockquote-footer">BMC Bioinformatics (2009)</footer>
    </blockquote>

    </div>

  </div>

<!--#include file="footer.hidden" -->
