<!--#include file="header.hidden" -->

<script>
  $("#link_software").css("background-color", "#cacaca");
</script>

  <div class="container">
    <br>
    <p class="font-weight-light">

    <i class="fas fa-file-code"></i>&nbsp;<strong>Software</strong>

    <br>List of research platforms I (co)-developed</a>

    <table border=0 style='margin-left: 10px;' class='font-weight-light'>
      <tr>
        <td style='padding-right: 10px; vertical-align:top'><a href="https://expressRNA.org" target="_new"><img src='/img/expressRNA_circle.png' width=40></a></td>
        <td><a href="https://expressRNA.org" target="_new">expressRNA</a>: post-transcriptional open-source integrative analytics platform, combining protein-RNA binding datasets with feature selection (from RNA-seq) and motif signals. I developed this platform during my Phd and i am keeping it alive, also using my own personal funding. Currently, we are extending it to allow upload of long-read sequencing data (Nanopore, PacBio). Additionally, we plan to include single-cell RNA-seq alternative polyadenylation analysis, since 10xGenomics data is 3'-end targeted, which allows for detailed studies of RNA 3'-end regulation.</td>
      </tr>
    </table>

    <br>

    <table border=0 style='margin-left: 10px;' class='font-weight-light'>
      <tr>
        <td style='padding-right: 10px; vertical-align:top'><a href="http://microbeAtlas.org" target="_new"><img src='/img/map.png' width=40></a></td>
        <td><a href="https://microbeatlas.org" target="_new">microbeAtlas</a>: Abundance and cluster analyses of more than 1 million metagenome samples. A distinguishing feature of our approach is that we do not only provide the analysis results of individual microbial communities or single studies, but combine the analysis of all microbial communities and studies to obtain a comprehensive picture of the ecology of microbes. This resource was made possible by the conclusion of several methods, tools and pipelines (<a href='https://github.com/jfmrod/hpc-clust' target=_new>HPC-CLUST</a> and <a href='https://github.com/jfmrod/MAPseq' target=_new>MAPseq</a>).</td>
      </tr>
    </table>

    <br>

    <table border=0 style='margin-left: 10px;' class='font-weight-light'>
      <tr>
        <td style='padding-right: 10px; vertical-align:top'><a href="https://github.com/grexor/rnamotifs2" target="_new"><img src='/img/rnamotifs2_circle.png' width=40></a></td>
        <td><a href="https://github.com/grexor/rnamotifs2" target="_new">RNAmotifs2</a>: an independent Python package, the cluster version of the RNAmotifs software (extended to look for regulatory motif clusters). The software is integrated with expressRNA and for now doesn't have a web interface of it's own.</td>
      </tr>
    </table>

    <br>

    <table border=0 style='margin-left: 10px;' class='font-weight-light'>
      <tr>
        <td style='padding-right: 10px; vertical-align:top'><a href="http://snpsyn.biolab.si/" target="_new"><img src='/img/snpsyn_circle.png' width=40></a></td>
        <td><a href="http://snpsyn.biolab.si/" target="_new">SNPsyn</a>: a Flash interface to the GWAS SNP-SNP interaction web analytics software. It allows the upload of SNP data with phenotype to then compute synergistic SNP pairs which explain the phenotype.</td>
      </tr>
    </table>

    <br>

    <table border=0 style='margin-left: 10px;' class='font-weight-light'>
      <tr>
        <td style='padding-right: 10px; vertical-align:top'><a href="http://dictyexpress.biolab.si/" target="_new"><img src='/img/dicty_circle.png' width=40></a></td>
        <td><a href="http://dictyexpress.biolab.si/" target="_new">dictyExpress</a>: gene expression profile bioinformatics analytics of <a href='https://en.wikipedia.org/wiki/Dictyostelium_discoideum'><i>Dictyostelium discoideum</i></a>, a very interesting (and social!) amoeba. The initial version of this software (Rot et al., 2009) was one of the first interfaces of this kind (Flash), which allowed exploratory bioinformatics analysis (gene expression profiles, clustering, GO enrichment, co-expression) from RNA-seq data.</td>
      </tr>
    </table>
  </div>

<!--#include file="footer.hidden" -->
